local/network.go

```diff
@@ -122,7 +122,45 @@ func init() {
 	if err != nil {
 		panic(err)
 	}
-	defaultNetworkConfig.Genesis = string(genesis)
+
+	// update genesis validation start time
+	var genesisMap map[string]interface{}
+	if err = json.Unmarshal(genesis, &genesisMap); err != nil {
+		panic(err)
+	}
+	startTime := time.Now().Unix()
+	lockTime := startTime + 2836800
```

Comment:
what's the magic number?

Reply:
It is the difference between lockTime and startTime in avago local genesis. But just did not asked on the reason behing it